.eslintrc.js

```diff
@@ -20,6 +20,8 @@ module.exports = {
     'no-unused-vars': 0,
     'no-extra-boolean-cast': 0,
     'no-inner-declarations': 0,
+    'no-constant-condition': 0,
+    'no-async-promise-executor': 0,
```

Comment:
Maybe remove this line? I don't see anywhere violates this rule

Reply:
it catch eslint error when
white(true)
vs
new Promise(async(resolve,reject)=>{...})

I see it too stupid :D but ok, I will remain it for u.

Reply:
Line 23 is needed, but line 24 is not, i don't see async promise in your changes

Reply:
updated, continue to review 2 project please